dev-tools/test.sh

```diff
@@ -18,19 +18,23 @@ export INTEGREAT_CMS_SUMM_AI_API_KEY="dummy"
 
 # Disable linkcheck listeners during testing
 export INTEGREAT_CMS_LINKCHECK_DISABLE_LISTENERS=1
+NUMPROCESSES="--numprocesses=auto"
 
 # Parse given command line arguments
 while [[ $# -gt 0 ]]; do
     case $1 in
         # If only tests affected by recent changed should be run, --changed can be passed as a flag
         --changed) CHANGED=1;shift;;
+        # Verbosity for pytest (one extra v to counteract --quiet)
+        # also disable multiple processes by xdist so diffs are full terminal width
+        -v|-vv|-vvv|-vvvv) VERBOSITY="$1v";NUMPROCESSES="";shift;;
```

Comment:
```suggestion
        # Verbosity for pytest
        -v|-vv|-vvv|-vvvv) VERBOSITY="$1";shift;;
```
Then we don't need to counteract `--quiet` here